dts/bindings/spi/microchip,xec-qmspi-ldma.yaml

```diff
@@ -0,0 +1,106 @@
+# Copyright (c) 2018, Google LLC.
+# Copyright (c) 2021, Microchip Technology Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+description: Microchip XEC QMSPI controller with local DMA
+
+compatible: "microchip,xec-qmspi-ldma"
+
+include: spi-controller.yaml
+
+properties:
+    reg:
+      required: true
+
+    girqs:
+      type: array
+      required: true
+      description: |
+        An array of integers encoding each interrupt signal connection.
+        This information includes the aggregated GIRQ number, GIRQ bit
+        position, aggregated GIRQ NVIC connection, and direct NVIC
+        connection of the GIRQ bit.
+
+    pcrs:
+      type: array
+      required: true
+      description: |
+        A two entry integer array containing the QMSPI PCR sleep register
+        index and bit position.
+
+    lines:
+      type: int
+      required: false
+      description: |
+        QMSPI data lines 1, 2, or 4. 1 data line is full-duplex
+        MOSI and MISO or half-duplex on MOSI only. Lines set to 2
+        or 4 indicate dual or quad I/O modes. Defaults to 1.
+
+    port_sel:
```

Comment:
Please use dashes (`-`), not underscores (`_`) for all properties

Reply:
Fixed with latest push

Reply:
@mbolivar-nordic dismissing the review to proceed with merge as suggested